src/features/category/ui/routes/CategoryRoute/CategoryRoute.tsx

```
@@ -0,0 +1,102 @@
+import { cn } from '@/shared/ui/styles/utils';
+import SubPageHeader from '@/shared/ui/SubPageHeader';
+import UnderlinedTextInput from '@/shared/ui/UnderlinedTextInput';
+
+const CategoryRoute: React.FC = () => {
+  const [input, setInput] = React.useState<string>('');
+
+  const form = useForm<{ categoryName: string }>({
+    defaultValues: { categoryName: '' },
+  });
+
+  const onSubmit = (values: { categoryName: string }) => {
```

ESLint

[error] 'values' is defined but never used.

(@typescript-eslint/no-unused-vars)